frontend/admin/src/js/components/user/GeneralInformationTab/GeneralInformationTab.tsx

```diff
@@ -0,0 +1,664 @@
+            {
+              defaultMessage:
+                "<purpleText>Open to opportunities</purpleText> - Not actively looking but still wants to be contacted for job opportunities",
+              description:
+                "Text in view user page saying they currently have the 'Open to opportunities' status, ignore things in <> tags please",
+            },
+            {
+              purpleText,
+            },
+          )}
+        {user.jobLookingStatus === JobLookingStatus.Inactive &&
+          intl.formatMessage(
+            {
+              defaultMessage:
+                "<purpleText>Inactive</purpleText> - Does not want to be contacted for job opportunities",
+              description:
+                "Text in view user page saying they currently have the 'Inactive' status, ignore things in <> tags please",
+            },
+            {
+              purpleText,
+            },
+          )}
+      </div>
+      <h3>
+        {intl.formatMessage({
+          defaultMessage: "Pool status",
+          description:
+            "Title of the 'Pool status' section of the view-user page",
+        })}
+      </h3>
+      <PoolStatusTable user={user} />
+      <h3>
+        {intl.formatMessage({
+          defaultMessage: "Add user to other pools",
+          description:
+            "Title of the 'Add user to pools' section of the view-user page",
+        })}
+      </h3>
+      <Button
+        color="primary"
+        mode="outline"
+        onClick={() => {
+          setShowAddToPoolDialog(true);
+        }}
+      >
+        <span data-h2-font-style="b(underline)">
+          {intl.formatMessage({
+            defaultMessage: "Add user to pool",
+            description: "Button to add user to pool on the view-user page",
+          })}
+        </span>
+      </Button>
+      <AddToPoolDialog
+        isVisible={showAddToPoolDialog}
+        user={user}
+        pools={pools}
+        onDismiss={() => setShowAddToPoolDialog(false)}
+      />
+    </>
+  );
+};
+
+const NotesSection: React.FC<BasicSectionProps> = ({ user }) => {
+  const intl = useIntl();
+  const locale = getLocale(intl);
+
+  const [, executeMutation] = useUpdatePoolCandidateMutation();
+
+  const handleUpdateCandidate = async (
+    id: string,
+    values: UpdatePoolCandidateAsAdminInput,
+  ) => {
+    const res = await executeMutation({ id, poolCandidate: values });
+    if (res.data?.updatePoolCandidateAsAdmin) {
+      return res.data.updatePoolCandidateAsAdmin;
+    }
+    return Promise.reject(res.error);
+  };
+
+  const handleSubmit = async (formValues: { [x: string]: string }) => {
+    user?.poolCandidates?.forEach(async (candidate) => {
+      if (candidate && (candidate.notes || "") !== formValues[candidate.id]) {
+        await handleUpdateCandidate(candidate.id, {
+          notes: formValues[candidate.id],
+        })
+          .then(() => {
+            toast.success(
+              intl
+                .formatMessage({
```

Comment:
Please use a parameterized formatMessge instead of a concat.  It's better to give the translator the whole message to work with.